### ISSUE TITLE
Inaccurate calculations in 'Part 5 - Inference and Validation (Exercises).ipynb'

### ISSUE THREAD

**Issue by HaleTom** [udacity/deep-learning-v2-pytorch#71]
In the test set, there are:

```
len(testloader.dataset)) = int(10000)
```

examples.

The `with torch.no_grad():` loop runs 157 times (10_000 / 64 examples per minibatch ==  156.25)

also, `len(testloader)) = int(157) `

The issue is with the line:
```
"Test Accuracy: {:.3f}".format(accuracy/len(testloader)))
```

This should be dividing by 156.25 not 157.
